src/command_modules/azure-cli-lab/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
++++++++++++++++++++++
+
+* Adding commands to manage secrets in the Azure DevTest Lab.
```

Comment:
Since these are the release notes FOR DevTestLab (instead of the CLI in general), I would recommend something like:

* Add commands to manage secrets within a lab.